File: week1/김한서/1676.js
const readline = require("readline");

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input = [];

rl.on("line", function (line) {
  input.push(line);
});

rl.on("close", function () {
  const n = Number(input[0]);
  let answer = 0;
  let num = n;

  // 5가 곱해진 횟수 세기
  while (num >= 5) {
    answer += Math.floor(num / 5);
    // 25, 125 같은 숫자들 고려
    num = Math.floor(num / 5);
  }

  console.log(answer);
});

File: week1/장수빈/1676.py
# 팩토리얼 0의 개수

import sys
from math import factorial

n = int(sys.stdin.readline()) 

# print(factorial(n))

# print(factorial(5))
# print(factorial(10))
# print(factorial(15))
# print(factorial(20))
# print(factorial(25))
# print(factorial(30))

print(n // 5 + n // 25 + n // 125)